backend/typescript/rest/entityRoutes.ts

```diff
@@ -49,23 +50,32 @@ entityRouter.post(
 
 /* Get all entities */
 entityRouter.get("/", async (_req, res) => {
+  const contentType = _req.headers["content-type"];
   try {
     const entities = await entityService.getEntities();
-    res.status(200).json(entities);
+    await setResponseByMimeType(res, 200, contentType, entities);
   } catch (e) {
-    res.status(500).send(e.message);
+    await setResponseByMimeType(res, 500, contentType, [
+      {
+        error: e.message,
+      },
+    ]);
   }
 });
 
 /* Get entity by id */
 entityRouter.get("/:id", async (req, res) => {
```

Comment:
Don't think we need to allow `text/csv` requests for GET requests for singular entities :)